packages/react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -364,7 +364,12 @@ class RCTAztecView: Aztec.TextView {
         let objectPlaceholder = "\u{FFFC}"
         let dictationText = dictationResult.reduce("") { $0 + $1.text }
         isInsertingDictationResult = false
-        self.text = self.text?.replacingOccurrences(of: objectPlaceholder, with: dictationText)
```

Comment:
While it's necessary to keep much of this code in place to avoid an unwanted `obj` symbol, replacing `self.text` in this way is causing issues. By directly calling the `insertText` function, we achieve the same functionality and avoid the empty string being propagated.